gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/converter/StringToAttributeKindConverterTest.java

```diff
@@ -0,0 +1,33 @@
+package org.hypertrace.gateway.service.common.converter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.hypertrace.core.attribute.service.v1.AttributeKind;
+import org.hypertrace.gateway.service.common.converters.StringToAttributeKindConverter;
+import org.hypertrace.gateway.service.v1.common.Value;
+import org.junit.jupiter.api.Test;
+
+public class StringToAttributeKindConverterTest {
+
+  @Test
+  public void test_defaultProtoEmptyString_ToNumerical_setToDefault() {
+    StringToAttributeKindConverter converter = StringToAttributeKindConverter.INSTANCE;
+    Value actualValue =
+        converter.doConvert("", AttributeKind.TYPE_DOUBLE, Value.newBuilder());
+    assertEquals(0.0d, actualValue.getDouble());
+
+    actualValue =
+        converter.doConvert("", AttributeKind.TYPE_DOUBLE, Value.newBuilder());
```

Comment:
Should the type here be TYPE_INT64 instead?

Reply:
Yes, fixed this